src/app/main.rs

```diff
@@ -276,7 +276,11 @@ where
                     .layer(http_metrics::layer::<_, classify::Response>(
                         ctl_http_metrics.clone(),
                     ))
-                    .layer(reconnect::layer().with_fixed_backoff(config.control_backoff_delay))
+                    .layer(reconnect::layer().with_exp_backoff(
+                        config.control_exp_backoff_min,
+                        config.control_exp_backoff_max,
+                        config.control_exp_backoff_max_jitter,
+                    ))
```

Comment:
and if we follow the above recommendation, all of these turn into the less-verbose 
```rust
reconnect::layer().with_backoff(config.control_backoff.clone())
```